src/compose/composeappmanager.cc

```diff
@@ -287,8 +276,20 @@ data::InstallationResult ComposeAppManager::install(const Uptane::Target& target
 
 data::InstallationResult ComposeAppManager::finalizeInstall(const Uptane::Target& target) {
   auto ir = OstreeManager::finalizeInstall(target);
-  if (ir.result_code != data::ResultCode::Numeric::kAlreadyProcessed ||
-      ir.result_code != data::ResultCode::Numeric::kNeedCompletion) {
+
+  const auto& current_apps = getApps(target);
```

Comment:
current_apps --> apps?

Reply:
"current" is pretty important context in many cases.